onnxruntime/test/providers/cpu/tensor/slice_op.test.cc

```diff
@@ -472,17 +472,15 @@ TEST(SliceTest, Slice3D_WithPositiveAndNegativeSteps_SubsetOfAxes_2) {
 }
 
 // Slice for Reversing
-// With numeric_limit_max, it means slice to the end of a dimension
-// (whichever direction we are stepping)
-TEST(SliceTest, Slice1D_ReverseAllAxes_1) {
+TEST(SliceTest, Slice1D_ReverseAllAxes_EmptyResult) {
   RunSliceTest<float>({4},
                       {1.0f, 2.0f, 3.0f, 4.0f},
                       {-1},
                       {std::numeric_limits<int32_t>::max()},
                       {0},
                       {-1},
-                      {4},
-                      {4.0f, 3.0f, 2.0f, 1.0f},
+                      {0},
+                      {},
                       true);
```

Comment:
If the opset is 10 shouldn't this preserve the old behavior?

Reply:
https://github.com/microsoft/onnxruntime/issues/11107#issuecomment-1090656720